File: abc231/b/a.cc
#include <iostream>
#include <map>
using namespace std;

int main() {
  int N;
  cin >> N;
  map<string, int> mp;
  for (int n = 0; n < N; n++) {
    string S;
    cin >> S;
    mp[S]++;
  }
  string ans;
  int best = -1;
  for (map<string, int>::iterator it = mp.begin(); it != mp.end(); it++) {
    if (best < it->second) {
      ans = it->first;
      best = it->second;
    }
  }
  cout << ans << endl;
  return 0;
}

File: abc231/b/a.py
N = int(input())
dict = {}
for n in range(N):
    S = input()
    if S in dict:
        dict[S] = dict[S] + 1
    else:
        dict[S] = 1
best = -1
ans = ''
for name, value in dict.items():
    if value > best:
        best = value
        ans = name
print(ans)
